group_vars/all.yml

```diff
@@ -83,12 +83,12 @@ girder_item_template_source:
               type: string
               index: not_analyzed
 
-mongo_url: localhost
-celery_broker: "mongodb://localhost:27017/tasks"
+mongo_url: 172.30.2.123
```

Comment:
I recommend overriding the localhost urls with specific ips in the docker.yml or prod.yml file.